lib/urlwatch/filters.py

```diff
@@ -260,7 +260,8 @@ def filter(self, data, subfilter):
         else:
             method = 're'
             options = {}
-
+        if method == 'pyhtml2text':
+            setattr(self.job, 'is_markdown', True)
```

Comment:
This can just be:

    self.job.is_markdown = True

No need to use `setattr()` here.